apps/rush-lib/src/cli/actions/ListAction.ts

```diff
@@ -1,76 +1,98 @@
-import { BaseRushAction } from './BaseRushAction';
-import { RushCommandLineParser } from '../RushCommandLineParser';
-import { CommandLineFlagParameter } from '@microsoft/ts-command-line';
-import { RushConfigurationProject } from '../../api/RushConfigurationProject';
-import * as Table from 'cli-table';
+import { BaseRushAction } from "./BaseRushAction";
```

Comment:
Revert these changes. We use single-quoted strings.

CI should have failed with a lint error here. @octogonz?